src/background/Wallet/model/WalletContainer.ts

```diff
@@ -165,6 +181,7 @@ export class MnemonicWalletContainer extends WalletContainerImpl {
       seedType === SeedType.mnemonic,
       'Must be a MnemonicWalletContainer'
     );
+    assertMnemonicWallets(wallets);
```

Comment:
Why do we need it now?

Reply:
The types gave warning all of a sudden, but this has always been implied